lib/hanami/cli/commands/app/generate/component.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "dry/inflector"
+require "dry/files"
+require "shellwords"
+module Hanami
+  module CLI
+    module Commands
+      module App
+        module Generate
+          # @api private
+          class Component < App::Command
+            argument :name, required: true, desc: "Component name"
+            option :slice, required: false, desc: "Slice name"
+
+            example [
+              %(operations.create_user               (MyApp::Operations::CreateUser)),
+              %(operations.user.create               (MyApp::Operations::Create::User)),
+              %(operations.create_user --slice=admin (Admin::Operations::CreateUser)),
+              %(Operations::CreateUser (MyApp::Operations::CreateUser)),
```

Comment:
Should this line be spaced out so `(MyApp::Operations::CreateUser)` aligns with the lines above?

Reply:
corrected the indentation